ReactNativeClient/lib/import-enex.js

```diff
@@ -306,7 +311,7 @@ function importEnex(parentFolderId, filePath, importOptions = null) {
 			if (n == 'note') {
 				note = {
 					resources: [],
-					tags: [],
+					tags: ['enex import', `enex to ${importOptions.outputFormat} import`],
```

Comment:
One thing I learnt with the Welcome notes fiasco, is that users don't like auto-created tags so I would prefer if we don't create these.